### PR TITLE
[MFTF]Add new ActionGroup 

### PR DESCRIPTION
**Pull request by Usik2203**
This PR created new `AdminClickButtonAddTaxRuleActionGroup` for clicking button Add New button for creating new Tax Rule

### CODE REVIEW

**Comment by Usik2203** [comment]
@magento run Functional Tests CE , Functional Tests EE , Functional Tests B2B

**Comment by Usik2203** [comment]
@magento run  Functional Tests B2B

File: app/code/Magento/Tax/Test/Mftf/ActionGroup/AdminClickButtonAddTaxRuleActionGroup.xml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="AdminClickButtonAddTaxRuleActionGroup">
```
Comment (eduard13) [inline app/code/Magento/Tax/Test/Mftf/ActionGroup/AdminClickButtonAddTaxRuleActionGroup.xml]:
```suggestion
    <actionGroup name="AdminClickAddTaxRuleButtonActionGroup">
```

**Review by eduard13 — CHANGES_REQUESTED** [review]
Could you rename the action group please?
Thank you.

**Review by eduard13 — CHANGES_REQUESTED** [review]
Trying to fix labels' assignment.

**Comment by Usik2203** [comment]
@magento run all tests

File: app/code/Magento/Tax/Test/Mftf/ActionGroup/AdminClickButtonAddTaxRuleActionGroup.xml (hunk 1 above)
Comment (Usik2203) [reply app/code/Magento/Tax/Test/Mftf/ActionGroup/AdminClickButtonAddTaxRuleActionGroup.xml]: **Done**

**Review by eduard13 — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @eduard13, thank you for the review.
[ENGCOM-8040](https://jira.corp.magento.com/browse/ENGCOM-8040) has been created to process this Pull Request

**Comment by engcom-Alfa** [comment]
QA not applicable

**Pull request merged**